apiserver/facades/agent/deployer/deployer.go

```diff
@@ -272,7 +268,7 @@ func (d *DeployerAPI) SetStatus(ctx context.Context, args params.SetStatus) (par
 // embedded APIAddresser *without* bumping the facade version.
 // It should be blanked when this facade version is next incremented.
 func (d *DeployerAPI) ModelUUID() params.StringResult {
-	return params.StringResult{Result: d.st.ModelUUID()}
+	return params.StringResult{Result: ""}
 }
```

Comment:
Do we have a note somewhere to re-implement this?

Reply:
No, the comments on that method say it's deprecated, just waiting for it to be removed on the next facade version bump.